_episodes/03-index-slice-subset.md

```diff
@@ -16,7 +16,10 @@ objectives:
     - "Locate subsets of data using masks."
     - "Describe BOOLEAN objects in Python and manipulate data using BOOLEANs."
 keypoints:
-    - "FIXME"
+    - "Data can be subset using column headings, numeric ranges , operators, or specific index locations."
+    - "Python uses 0-based indexing."
```

Comment:
I suggest either adding a brief explanation or replacing the keypoint with it.
```
Python uses 0-based indexing: <brief explanation>
```
Explanation could be:
```
The first element in a list, tuple or any other data structure has an index of 0
```

Reply:
```suggestion
    - "Python uses 0-based indexing, in which the first element in a list, tuple or any other data structure has an index of 0."
```

Reply:
thanks april! suggestion incorporated in https://github.com/datacarpentry/python-ecology-lesson/pull/322/commits/034f56aaad5f0332b02d7d7cd1d38238ba8d1fc0